### ISSUE TITLE
Instance vs. Batch Normalization Mistake

### ISSUE THREAD

**Issue by ethanharvey98** [NYUMedML/CNN_design_for_AD#7]
Hi,

It looks to me like there is a mistake in [models/models.py](https://github.com/NYUMedML/CNN_design_for_AD/blob/master/models/models.py). When `norm_type` is `"Instance"` there 4 instance normalization layers, however, when `norm_type` is not `"Instance"` there are only 3 batch normalization layers. This is because L100 reads `self.conv.add_module('lrn2_s1',nn.BatchNorm3d(64*expansion))` instead of `self.conv.add_module('lrn3_s1',nn.BatchNorm3d(64*expansion))`. I can imagine that this could have a significant impact on the finding that "(1) instance normalization outperforms batch normalization".

```python
 87        if norm_type == 'Instance':
 88            self.conv.add_module('lrn2_s1',nn.InstanceNorm3d(64*expansion))
 89        else:
 90            self.conv.add_module('lrn2_s1',nn.BatchNorm3d(64*expansion))
```
...
```python
 97        if norm_type == 'Instance':
 98            self.conv.add_module('lrn3_s1',nn.InstanceNorm3d(64*expansion))
 99        else:
100            self.conv.add_module('lrn2_s1',nn.BatchNorm3d(64*expansion))
```

It also looks like there is a similar mistake in L102 which reads `self.conv.add_module('pool2_s1',nn.MaxPool3d(kernel_size=5, stride=2))` instead of `self.conv.add_module('pool3_s1',nn.MaxPool3d(kernel_size=5, stride=2))`. As is, I don't think the current implementation matches Table 2 in the paper.
